README.md

```diff
@@ -58,64 +63,14 @@ var contents = File.ReadAllText("01-GEN.usfm");
 USFMDocument output = parser.ParseFromString(contents);
 ```
 
-## HtmlRenderer
-
-This class transforms a USFMDocument into an html string
-
-Example:
-```csharp
-var contents = File.ReadAllText("01-GEN.usfm");
-USFMDocument document = parser.ParseFromString(contents);
-string html = renderer.Render(document);
-File.WriteAllText("output.html", html);
-
-```
-# What tags are supported
-
-Eventially the goal is to support the whole USFM spec but for the moment the following common tags are implemented
-
-## By the parser
```

Comment:
I was thinking it would get really long. If it's just linear, it looks unappealing. I was thinking of making a wiki page on github with a table of implemented markers :)

Reply:
Yeah that works.